docs/metrics.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[metrics]]
 == Metrics
 
-The Go agent periodically gathers and reports metrics. You can control how
+The Go agent periodically gathers and reports metrics. Control how
```

Comment:
Go {agent}?

Reply:
@leahleahy, `{agent}` resolves to `Elastic Agent`, which is different from the APM agents. In the APM agent docs, we use just the word, `agent`.

Reply:
Got it, thank you!